### ISSUE TITLE
Future of this package

### ISSUE THREAD

**Issue by Evizero** [tbreloff/MLPlots.jl#13]
A lot of recipes have now moved to their respective packages. 

What do we want to do with this?

**Comment by tbreloff** [comment]
Not sure. It might be good to keep it as a catch-all for JuliaML plotting.
Or maybe everything has a better home?

**Comment by Evizero** [comment]
Well one thing we could do here is test the recipes in our JuliaML packages. As it is currently I avoid testing those in the main packages because of travis inconveniences.

**Comment by Evizero** [comment]
Either way, this package would need some cleanup. For example `subplot` is still used here, while I think that doesn't exist anymore in Plots

**Comment by tbreloff** [comment]
Agreed. This needs cleanup. I like your idea to do all the testing here! (I
really need to improve VRT)
